src/clone.c

```diff
@@ -833,13 +833,13 @@ FuncDeclaration *buildPostBlit(StructDeclaration *sd, Scope *sc)
             ea = new CastExp(loc, ea, Type::tvoid->pointerTo());
 
             Expression *et = getTypeInfo(v->type, sc);
-            et = new DotIdExp(loc, et, Id::postblit);
+            et = new DotIdExp(loc, et, Identifier::idPool("postblit"));
```

Comment:
I know, but I can only have one `Id::postblit`, and for the sake of consistency I made `Id::postblit` like `Id::ctor` to mean the native mangling.